### ISSUE TITLE
Proton 6.8 and higher: ntdll.so - No such file or directory

### ISSUE THREAD

**Issue by killstrelok** [Matoking/protontricks#107]
Hi!

It appears with GlouriousEggroll's proton versions above 6.8 protontricks fails to point at the new location of libraries, example:
> \>protontricks 244210 -q dotnet48
warning: /home/user/.cache/protontricks/proton/Proton-6.10-GE-1/bin/wine cmd.exe /c echo '%AppData%' returned empty string, error message "wine: could not load ntdll.so: /home/user/.steam/root/compatibilitytools.d/Proton-6.10-GE-1/files/lib/wine/ntdll.so: cannot open shared object file: No such file or directory"

I believe this might have something to do with the fact that there's been a new directory structure introduced which breaks the compatibility as in 6.8+ the path to the ntdll.so file contains one more directory: ~/.steam/root/compatibilitytools.d/Proton-6.10-GE-1/files/lib/wine/**i386-unix**/ntdll.so

Protontricks version: 1.5.1
Winetricks version: 20210206-next
OS: Ubuntu 18.04

BR

**Comment by Matoking** [comment]
This looks like it should be fixed by 7fd1f619d149805b92ed767bddc6d30b0d9facb2 that is in master branch but hasn't been included in a proper release yet, though I'll look into making one soon.

For now, could you try installing the development version and check if the issue still occurs? Since you're running Ubuntu, you might have installed Protontricks using pipx. In that case, the following should do the trick:

```
$ pipx install --force git+https://github.com/Matoking/protontricks.git
# or try this if the previous command does not work
$ pipx install --force --spec git+https://github.com/Matoking/protontricks.git protontricks
```

**Comment by killstrelok** [comment]
Can confirm, this works! Thank you for your help :)

**Comment by Matoking** [comment]
Thanks for the quick confirmation.

I've released 1.5.2, so the fix should hopefully be available for distro packages soon as well.
